### DISCUSSION TITLE
Retrieve an array stored in Alpine.store

### DISCUSSION THREAD

**Discussion by 24glinka** [alpinejs/alpine#2439] [4. General]
Hi, I have a simple question: I would like to retrieve an object in Alpine.store, this object is an array of objects.
When I'm printing in the console what is stored, I see something like this:

`Proxy {<target>: Array(848) [ {…}, {…}, {…}, … ], <handler>: Object { get: ke(n, i, o), set: Cr(r, n, i, o), deleteProperty: Fn(e, t), … }}`

The objects I would like to get is inside <target> and I would like to do it inside a Javascript file.
How can I do that? I tried a direct access as well as using an internal function but with the same result.

**Comment by SimoTod** [comment]
Don't print it, just use it as normal. Alpine objects are wrapped in javascript proxies to implement reactivity.

**Reply by 24glinka** [reply]
Ok, issue solved: thanks!

**Comment by elfeffe** [comment]
How to access the array? I can't use foreach with the array stored inside an AlpineJS store.

**Reply by 24glinka** [reply]
The array is visible simply by getting it from Alpine store.
